tests/test_proxy_info.py

```diff
@@ -45,51 +45,54 @@
 local_ota_proxy_listen_port: "2808"
 """
 
-# check ProxyInfo for detail
-_COMMON_DEFAULT_PARSED_CONFIGS: Dict[str, Any] = {
-    "enable_local_ota_proxy": True,
+# NOTE: check docs/README.md for details
+DEFAULT_SETTINGS_FOR_PROXY_INFO = {
```

Comment:
This naming seems to be misleading I think.
If the proxy_info.yaml doesn't exist, default setting is used, but this dict is not that of `default setting`.

Reply:
Updated at [fc68016](https://github.com/tier4/ota-client/pull/187/commits/fc68016e62cdd9fe141785ad6470b58d35ba8a5e).

@obi-t4 Now `default setting` and `proxy_info.yaml missing/corrupted` are unrelated different 2 things.
1. when `proxy_info.yaml missing/corrupted`, a `PRE_DEFINED_PROXY_INFO_YAML` will be used, the `PRE_DEFINED_PROXY_INFO_YAML` is as follow: 
    ```yaml
    enable_local_ota_proxy: true
    gateway: true
    ```
3. `default setting` is the default value for each config field when field is not specified/invalid in `proxy_info.yaml`.